### PR TITLE
Implementing testing framework mocha and assertion library chai.

### PR DESCRIPTION
**Pull request by akl2390**
Unit testing frameworks implemented, sample files are included.

The unit tests for our own files have not been completed.

### CODE REVIEW

File: grr-ui/src/js/main.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
+    function sayHello(){
```
Comment (ask7708) [inline grr-ui/src/js/main.js]: Is this relevant to be adding to our main code?

Comment (akl2390) [reply grr-ui/src/js/main.js]: nope, left it in by accident

File: grr-ui/Test/testSampleTest.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+const assert = require('chai').assert;
+const app = require('../src/js/testSample');
+
+//Results
+sayHelloResult = app.sayHello();
+addNumbersResult = app.addNumbers(5,5);
+
+describe('App', function(){
```
Comment (ask7708) [inline grr-ui/Test/testSampleTest.js]: How are you going to test the actual functions we have? The sample stuff looks nice but it'd be better if we at least had a test example for one of our current functions.

File: grr-ui/src/js/testSample.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
```
Comment (ask7708) [inline grr-ui/src/js/testSample.js]: How will you be able to export the functions we're using in our application for testing?

Comment (akl2390) [reply grr-ui/src/js/testSample.js]: There are a few ways to do this, but the easiest way looks like we should break our (to be tested) functions into "bite size" pieces that are safe for exporting. We can structure our app to make this easier, or just manually break the functions ourselves. I can make that a new story to set up everything and write unit tests for our stuff to see how that works out.

Comment (ask7708) [reply grr-ui/src/js/testSample.js]: Yeah I'd like to see an example of what you want to do in terms of testing those functions before we edit the code. Shouldn't there be a way to export them already?
